webpack.release.config.js

```diff
@@ -53,4 +55,16 @@ var devConfig = config('okta-sign-in.js', 'development');
 devConfig.plugins = plugins({ isProduction: false, analyzerFile: 'okta-sign-in.analyzer' });
 usePolyfill(devConfig);
 
-module.exports = [entryConfig, noPolyfillConfig, cdnConfig, noJqueryConfig, devConfig];
+// 6. plugins: a11y
+var pluginConfig = config('okta-plugins.js', 'production');
```

Comment:
how about name it as `okta-plugins.beta.js` so we have the freedom to make changes?

Reply:
Is there another way we can make clear it's experimental? Maybe a big disclaimer in the header? Changing the URL later will mean updating usage

Reply:
I feel it's ok to leave it in the filename/url, devs also need to make changes anyway if they are using a npm package with beta tag.
what the main use case for this plugin file? if it's mostly for monolith, then we still have the full control of the url.

Reply:
I primarily had the monolith scenario in mind during implementation, but AFAICT there's no reason it couldn't be used outside.